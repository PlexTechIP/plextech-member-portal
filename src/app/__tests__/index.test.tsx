import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import { App } from '../index';

const renderer = createRenderer();

describe('<App />', () => {});
