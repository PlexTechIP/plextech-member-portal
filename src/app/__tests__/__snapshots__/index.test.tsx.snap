// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render and match the snapshot 1`] = `
<React.Fragment>
  <ThemeProvider
    theme={
      {
        "breakpoints": {
          "between": [Function],
          "down": [Function],
          "keys": [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ],
          "not": [Function],
          "only": [Function],
          "unit": "px",
          "up": [Function],
          "values": {
            "lg": 1200,
            "md": 900,
            "sm": 600,
            "xl": 1536,
            "xs": 0,
          },
        },
        "components": {
          "MuiButton": {
            "styleOverrides": {
              "root": {
                "&:hover": {
                  "backgroundColor": "#eeeeee",
                },
                "backgroundColor": "transparent",
                "color": "rgb(255, 138, 0)",
                "fontWeight": "bold",
              },
            },
          },
        },
        "direction": "ltr",
        "mixins": {
          "toolbar": {
            "@media (min-width:0px)": {
              "@media (orientation: landscape)": {
                "minHeight": 48,
              },
            },
            "@media (min-width:600px)": {
              "minHeight": 64,
            },
            "minHeight": 56,
          },
        },
        "palette": {
          "action": {
            "activatedOpacity": 0.12,
            "active": "rgba(0, 0, 0, 0.54)",
            "disabled": "rgba(0, 0, 0, 0.26)",
            "disabledBackground": "rgba(0, 0, 0, 0.12)",
            "disabledOpacity": 0.38,
            "focus": "rgba(0, 0, 0, 0.12)",
            "focusOpacity": 0.12,
            "hover": "rgba(0, 0, 0, 0.04)",
            "hoverOpacity": 0.04,
            "selected": "rgba(0, 0, 0, 0.08)",
            "selectedOpacity": 0.08,
          },
          "augmentColor": [Function],
          "background": {
            "default": "#fff",
            "paper": "#fff",
          },
          "common": {
            "black": "#000",
            "white": "#fff",
          },
          "contrastThreshold": 3,
          "divider": "rgba(0, 0, 0, 0.12)",
          "error": {
            "contrastText": "#fff",
            "dark": "#c62828",
            "light": "#ef5350",
            "main": "#d32f2f",
          },
          "getContrastText": [Function],
          "grey": {
            "100": "#f5f5f5",
            "200": "#eeeeee",
            "300": "#e0e0e0",
            "400": "#bdbdbd",
            "50": "#fafafa",
            "500": "#9e9e9e",
            "600": "#757575",
            "700": "#616161",
            "800": "#424242",
            "900": "#212121",
            "A100": "#f5f5f5",
            "A200": "#eeeeee",
            "A400": "#bdbdbd",
            "A700": "#616161",
          },
          "info": {
            "contrastText": "#fff",
            "dark": "#01579b",
            "light": "#03a9f4",
            "main": "#0288d1",
          },
          "mode": "light",
          "primary": {
            "contrastText": "#fff",
            "dark": "#1565c0",
            "light": "#42a5f5",
            "main": "#1976d2",
          },
          "secondary": {
            "contrastText": "#fff",
            "dark": "#7b1fa2",
            "light": "#ba68c8",
            "main": "#9c27b0",
          },
          "success": {
            "contrastText": "#fff",
            "dark": "#1b5e20",
            "light": "#4caf50",
            "main": "#2e7d32",
          },
          "text": {
            "disabled": "rgba(0, 0, 0, 0.38)",
            "primary": "rgba(0, 0, 0, 0.87)",
            "secondary": "rgba(0, 0, 0, 0.6)",
          },
          "tonalOffset": 0.2,
          "warning": {
            "contrastText": "#fff",
            "dark": "#e65100",
            "light": "#ff9800",
            "main": "#ed6c02",
          },
        },
        "shadows": [
          "none",
          "0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12)",
          "0px 3px 1px -2px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 1px 5px 0px rgba(0,0,0,0.12)",
          "0px 3px 3px -2px rgba(0,0,0,0.2),0px 3px 4px 0px rgba(0,0,0,0.14),0px 1px 8px 0px rgba(0,0,0,0.12)",
          "0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12)",
          "0px 3px 5px -1px rgba(0,0,0,0.2),0px 5px 8px 0px rgba(0,0,0,0.14),0px 1px 14px 0px rgba(0,0,0,0.12)",
          "0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12)",
          "0px 4px 5px -2px rgba(0,0,0,0.2),0px 7px 10px 1px rgba(0,0,0,0.14),0px 2px 16px 1px rgba(0,0,0,0.12)",
          "0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12)",
          "0px 5px 6px -3px rgba(0,0,0,0.2),0px 9px 12px 1px rgba(0,0,0,0.14),0px 3px 16px 2px rgba(0,0,0,0.12)",
          "0px 6px 6px -3px rgba(0,0,0,0.2),0px 10px 14px 1px rgba(0,0,0,0.14),0px 4px 18px 3px rgba(0,0,0,0.12)",
          "0px 6px 7px -4px rgba(0,0,0,0.2),0px 11px 15px 1px rgba(0,0,0,0.14),0px 4px 20px 3px rgba(0,0,0,0.12)",
          "0px 7px 8px -4px rgba(0,0,0,0.2),0px 12px 17px 2px rgba(0,0,0,0.14),0px 5px 22px 4px rgba(0,0,0,0.12)",
          "0px 7px 8px -4px rgba(0,0,0,0.2),0px 13px 19px 2px rgba(0,0,0,0.14),0px 5px 24px 4px rgba(0,0,0,0.12)",
          "0px 7px 9px -4px rgba(0,0,0,0.2),0px 14px 21px 2px rgba(0,0,0,0.14),0px 5px 26px 4px rgba(0,0,0,0.12)",
          "0px 8px 9px -5px rgba(0,0,0,0.2),0px 15px 22px 2px rgba(0,0,0,0.14),0px 6px 28px 5px rgba(0,0,0,0.12)",
          "0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12)",
          "0px 8px 11px -5px rgba(0,0,0,0.2),0px 17px 26px 2px rgba(0,0,0,0.14),0px 6px 32px 5px rgba(0,0,0,0.12)",
          "0px 9px 11px -5px rgba(0,0,0,0.2),0px 18px 28px 2px rgba(0,0,0,0.14),0px 7px 34px 6px rgba(0,0,0,0.12)",
          "0px 9px 12px -6px rgba(0,0,0,0.2),0px 19px 29px 2px rgba(0,0,0,0.14),0px 7px 36px 6px rgba(0,0,0,0.12)",
          "0px 10px 13px -6px rgba(0,0,0,0.2),0px 20px 31px 3px rgba(0,0,0,0.14),0px 8px 38px 7px rgba(0,0,0,0.12)",
          "0px 10px 13px -6px rgba(0,0,0,0.2),0px 21px 33px 3px rgba(0,0,0,0.14),0px 8px 40px 7px rgba(0,0,0,0.12)",
          "0px 10px 14px -6px rgba(0,0,0,0.2),0px 22px 35px 3px rgba(0,0,0,0.14),0px 8px 42px 7px rgba(0,0,0,0.12)",
          "0px 11px 14px -7px rgba(0,0,0,0.2),0px 23px 36px 3px rgba(0,0,0,0.14),0px 9px 44px 8px rgba(0,0,0,0.12)",
          "0px 11px 15px -7px rgba(0,0,0,0.2),0px 24px 38px 3px rgba(0,0,0,0.14),0px 9px 46px 8px rgba(0,0,0,0.12)",
        ],
        "shape": {
          "borderRadius": 4,
        },
        "spacing": [Function],
        "transitions": {
          "create": [Function],
          "duration": {
            "complex": 375,
            "enteringScreen": 225,
            "leavingScreen": 195,
            "short": 250,
            "shorter": 200,
            "shortest": 150,
            "standard": 300,
          },
          "easing": {
            "easeIn": "cubic-bezier(0.4, 0, 1, 1)",
            "easeInOut": "cubic-bezier(0.4, 0, 0.2, 1)",
            "easeOut": "cubic-bezier(0.0, 0, 0.2, 1)",
            "sharp": "cubic-bezier(0.4, 0, 0.6, 1)",
          },
          "getAutoHeightDuration": [Function],
        },
        "typography": {
          "body1": {
            "fontFamily": ""DM Sans"",
            "fontSize": "1rem",
            "fontWeight": 400,
            "lineHeight": 1.5,
          },
          "body2": {
            "fontFamily": ""DM Sans"",
            "fontSize": "0.875rem",
            "fontWeight": 400,
            "lineHeight": 1.43,
          },
          "button": {
            "fontFamily": ""DM Sans"",
            "fontSize": "0.875rem",
            "fontWeight": 500,
            "lineHeight": 1.75,
            "textTransform": "uppercase",
          },
          "caption": {
            "fontFamily": ""DM Sans"",
            "fontSize": "0.75rem",
            "fontWeight": 400,
            "lineHeight": 1.66,
          },
          "fontFamily": ""DM Sans"",
          "fontSize": 14,
          "fontWeightBold": 700,
          "fontWeightLight": 300,
          "fontWeightMedium": 500,
          "fontWeightRegular": 400,
          "h1": {
            "fontFamily": ""DM Sans"",
            "fontSize": "6rem",
            "fontWeight": 300,
            "lineHeight": 1.167,
          },
          "h2": {
            "fontFamily": ""DM Sans"",
            "fontSize": "3.75rem",
            "fontWeight": 300,
            "lineHeight": 1.2,
          },
          "h3": {
            "fontFamily": ""DM Sans"",
            "fontSize": "3rem",
            "fontWeight": 400,
            "lineHeight": 1.167,
          },
          "h4": {
            "fontFamily": ""DM Sans"",
            "fontSize": "2.125rem",
            "fontWeight": 400,
            "lineHeight": 1.235,
          },
          "h5": {
            "fontFamily": ""DM Sans"",
            "fontSize": "1.5rem",
            "fontWeight": 400,
            "lineHeight": 1.334,
          },
          "h6": {
            "fontFamily": ""DM Sans"",
            "fontSize": "1.25rem",
            "fontWeight": 500,
            "lineHeight": 1.6,
          },
          "htmlFontSize": 16,
          "inherit": {
            "fontFamily": "inherit",
            "fontSize": "inherit",
            "fontWeight": "inherit",
            "letterSpacing": "inherit",
            "lineHeight": "inherit",
          },
          "overline": {
            "fontFamily": ""DM Sans"",
            "fontSize": "0.75rem",
            "fontWeight": 400,
            "lineHeight": 2.66,
            "textTransform": "uppercase",
          },
          "pxToRem": [Function],
          "subtitle1": {
            "fontFamily": ""DM Sans"",
            "fontSize": "1rem",
            "fontWeight": 400,
            "lineHeight": 1.75,
          },
          "subtitle2": {
            "fontFamily": ""DM Sans"",
            "fontSize": "0.875rem",
            "fontWeight": 500,
            "lineHeight": 1.57,
          },
        },
        "unstable_sx": [Function],
        "unstable_sxConfig": {
          "alignContent": {},
          "alignItems": {},
          "alignSelf": {},
          "backgroundColor": {
            "themeKey": "palette",
            "transform": [Function],
          },
          "bgcolor": {
            "cssProperty": "backgroundColor",
            "themeKey": "palette",
            "transform": [Function],
          },
          "border": {
            "themeKey": "borders",
            "transform": [Function],
          },
          "borderBottom": {
            "themeKey": "borders",
            "transform": [Function],
          },
          "borderBottomColor": {
            "themeKey": "palette",
          },
          "borderColor": {
            "themeKey": "palette",
          },
          "borderLeft": {
            "themeKey": "borders",
            "transform": [Function],
          },
          "borderLeftColor": {
            "themeKey": "palette",
          },
          "borderRadius": {
            "style": [Function],
            "themeKey": "shape.borderRadius",
          },
          "borderRight": {
            "themeKey": "borders",
            "transform": [Function],
          },
          "borderRightColor": {
            "themeKey": "palette",
          },
          "borderTop": {
            "themeKey": "borders",
            "transform": [Function],
          },
          "borderTopColor": {
            "themeKey": "palette",
          },
          "bottom": {},
          "boxShadow": {
            "themeKey": "shadows",
          },
          "boxSizing": {},
          "color": {
            "themeKey": "palette",
            "transform": [Function],
          },
          "columnGap": {
            "style": [Function],
          },
          "display": {},
          "displayPrint": {
            "cssProperty": false,
            "transform": [Function],
          },
          "flex": {},
          "flexBasis": {},
          "flexDirection": {},
          "flexGrow": {},
          "flexShrink": {},
          "flexWrap": {},
          "fontFamily": {
            "themeKey": "typography",
          },
          "fontSize": {
            "themeKey": "typography",
          },
          "fontStyle": {
            "themeKey": "typography",
          },
          "fontWeight": {
            "themeKey": "typography",
          },
          "gap": {
            "style": [Function],
          },
          "gridArea": {},
          "gridAutoColumns": {},
          "gridAutoFlow": {},
          "gridAutoRows": {},
          "gridColumn": {},
          "gridRow": {},
          "gridTemplateAreas": {},
          "gridTemplateColumns": {},
          "gridTemplateRows": {},
          "height": {
            "transform": [Function],
          },
          "justifyContent": {},
          "justifyItems": {},
          "justifySelf": {},
          "left": {},
          "letterSpacing": {},
          "lineHeight": {},
          "m": {
            "style": [Function],
          },
          "margin": {
            "style": [Function],
          },
          "marginBlock": {
            "style": [Function],
          },
          "marginBlockEnd": {
            "style": [Function],
          },
          "marginBlockStart": {
            "style": [Function],
          },
          "marginBottom": {
            "style": [Function],
          },
          "marginInline": {
            "style": [Function],
          },
          "marginInlineEnd": {
            "style": [Function],
          },
          "marginInlineStart": {
            "style": [Function],
          },
          "marginLeft": {
            "style": [Function],
          },
          "marginRight": {
            "style": [Function],
          },
          "marginTop": {
            "style": [Function],
          },
          "marginX": {
            "style": [Function],
          },
          "marginY": {
            "style": [Function],
          },
          "maxHeight": {
            "transform": [Function],
          },
          "maxWidth": {
            "style": [Function],
          },
          "mb": {
            "style": [Function],
          },
          "minHeight": {
            "transform": [Function],
          },
          "minWidth": {
            "transform": [Function],
          },
          "ml": {
            "style": [Function],
          },
          "mr": {
            "style": [Function],
          },
          "mt": {
            "style": [Function],
          },
          "mx": {
            "style": [Function],
          },
          "my": {
            "style": [Function],
          },
          "order": {},
          "overflow": {},
          "p": {
            "style": [Function],
          },
          "padding": {
            "style": [Function],
          },
          "paddingBlock": {
            "style": [Function],
          },
          "paddingBlockEnd": {
            "style": [Function],
          },
          "paddingBlockStart": {
            "style": [Function],
          },
          "paddingBottom": {
            "style": [Function],
          },
          "paddingInline": {
            "style": [Function],
          },
          "paddingInlineEnd": {
            "style": [Function],
          },
          "paddingInlineStart": {
            "style": [Function],
          },
          "paddingLeft": {
            "style": [Function],
          },
          "paddingRight": {
            "style": [Function],
          },
          "paddingTop": {
            "style": [Function],
          },
          "paddingX": {
            "style": [Function],
          },
          "paddingY": {
            "style": [Function],
          },
          "pb": {
            "style": [Function],
          },
          "pl": {
            "style": [Function],
          },
          "position": {},
          "pr": {
            "style": [Function],
          },
          "pt": {
            "style": [Function],
          },
          "px": {
            "style": [Function],
          },
          "py": {
            "style": [Function],
          },
          "right": {},
          "rowGap": {
            "style": [Function],
          },
          "textAlign": {},
          "textOverflow": {},
          "textTransform": {},
          "top": {},
          "typography": {
            "cssProperty": false,
            "themeKey": "typography",
          },
          "visibility": {},
          "whiteSpace": {},
          "width": {
            "transform": [Function],
          },
          "zIndex": {
            "themeKey": "zIndex",
          },
        },
        "zIndex": {
          "appBar": 1100,
          "drawer": 1200,
          "fab": 1050,
          "mobileStepper": 1000,
          "modal": 1300,
          "snackbar": 1400,
          "speedDial": 1050,
          "tooltip": 1500,
        },
      }
    }
  >
    <BrowserRouter>
      <Helmet
        defaultTitle="PlexTech Member Portal"
        defer={true}
        encodeSpecialCharacters={true}
        htmlAttributes={
          {
            "lang": undefined,
          }
        }
        prioritizeSeoTags={false}
        titleTemplate="%s - PlexTech Member Portal"
      >
        <meta
          content="PlexTech Member Portal"
          name="description"
        />
      </Helmet>
      <styled.div
        style={
          {
            "backgroundImage": "url(./shapes-background.png)",
          }
        }
      >
        <React.Fragment>
          <TopBar
            isDarkMode={false}
            open={false}
            setIsDarkMode={[Function]}
            setOpen={[Function]}
          />
          <Routes>
            <Route
              element={<Unknown />}
              path="/"
            />
            <Route
              element={<Unknown />}
              path="/reimbursements"
            />
            <Route
              element={<Unknown />}
              path="/attendance"
            />
            <Route
              element={<Unknown />}
              path="/forum"
            />
            <Route
              element={<Unknown />}
              path="*"
            />
          </Routes>
        </React.Fragment>
      </styled.div>
      <Memo(l) />
    </BrowserRouter>
  </ThemeProvider>
  <Analytics />
</React.Fragment>
`;
